### PR TITLE
Flowchart readme

### PR DESCRIPTION
**Pull request by ruudvdent**
added flowchart to the readme. I think this is a nice chart made by Vincent.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -23,9 +23,12 @@ Chris Weijenborg (Wageningen University)
 Peter Kalverla (Netherlands eScienceCenter)
 
 # How to use
-See the [documentation](http://wam2layers.rtfd.io/) for a more detailed description.
+See the [documentation](http://wam2layers.rtfd.io/) for a more detailed description. Still questions? Follow the flowchart below.
```
Comment (Imme1992) [inline README.md:26]: Change link to documentation to: https://wam2layers.readthedocs.io/en/latest/

File: misc/FlowChart GitHub.pptx
Comment (Imme1992) [inline misc/FlowChart GitHub.pptx:1]: Very nice the flowchart, thanks Vincent! Can you make the name of WAM-2Layers consistent to WAM2layers? Thanks!

Comment (vincentdefeiter) [reply misc/FlowChart GitHub.pptx:1]: @Imme1992 sure thing. Do you decide upon the name being WAM2Layers or WAM-2Layers, if so, I can adapt this everywhere.

Comment (Imme1992) [reply misc/FlowChart GitHub.pptx:1]: @vincentdefeiter Yes we decided on WAM2layers (notice the small letter l, similar as in the Github repository)

**Review by Imme1992 — APPROVED** [review]

**Pull request merged**